scripts/in_container/install_airflow_and_providers.py

```diff
@@ -754,14 +763,59 @@ def compile_ui_assets(
         cwd=os.fspath(source_ui_directory),
     )
     # copy compiled assets to installation directory
-    dist_source_directory = source_ui_directory / "dist"
+    dist_source_directory = source_ui_directory / UI_DIST_DIR_NAME
     console.print(
         f"[bright_blue]Copying compiled UI assets from '{dist_source_directory}' to '{dist_directory}'"
     )
     shutil.copytree(dist_source_directory, dist_directory)
     console.print("[bright_blue]UI assets compiled successfully")
 
 
+def copy_mounted_ui_dist(mounted_ui_dist_source: Path, dist_prefix: str, host_source_prefix: str):
```

Comment:
Hmm. I thinl rather than doing this copy, we could simply mount the folder directly where it is expected to be? I don't think we need to use the temporary folder to copy it from there inside?

Or am I wrong :) ?

Reply:
Hmm. I think I was wrong actually. When we do `--use-airflow-version`, airflow is installed after entering the docker container - in the entrypoint script, so mounting it would not work. The way it's implemented now is good.

Reply:
But.. on the other hand... actually... dist folder is ignored in git, so .. it will actually work :)